app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+from app.players.elves.elf_ranger import ElfRanger
+from app.players.elves.druid import Druid
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    total_rating = sum(player.get_rating() for player in players)
+    return total_rating
```

Comment:
There is no need for the `total_rating` variable, return the result of `sum` directly.